content/blog/asyncapi_codegen_scst.md

```diff
@@ -0,0 +1,142 @@
+---
+type: blog
+featured: true
+cover: /images/posts/asyncapi-codegen_pic-00.webp
+weight: 100
+title: "AsyncAPI Code Generation: Microservices Using Spring Cloud Stream"
+date: 2020-04-28 17:39:34 UTC
+tags:
+   - solace
+   - asyncapi
+   - java
+   - spring
+   - EDA
+authors:
+   - name: Marc DiPasquale
+     photo: /images/avatars/marc-dipasquale.webp
```

Comment:
you added new avatar, 68kb, pretty big ->  static/images/avatars/mdipasquale.jpg 
I recommend `webp` and make sure you reference it here as now it looks broken -> https://deploy-preview-106--asyncapi-website.netlify.app/blog/asyncapi_codegen_scst/

Reply:
done!